packages/common/src/types/payment/DonationHook.ts

```diff
@@ -0,0 +1,8 @@
+export interface IDonationHook {
+	service: string;
+	userId: string;
+	wishCardId: string;
+	amount: number;
+	userDonation: number | null;

```

Comment:
can it be ```null``` or ```undefined```? if undefined you can write it like so ```userDonation?: number;```

Reply:
Unfortunately, no since it's fed into stripe's API, which requires null.